### PR TITLE
DOCS-2027 Jira integration

### PR DESCRIPTION
**Pull request by MaximBashurov**

### CODE REVIEW

File: docs/latest/user-guides/settings/integrations/jira.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#   Jira
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/settings/integrations/jira.md:1]: what are these huge distances between # and a header name for?

File: docs/latest/user-guides/settings/integrations/jira.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+You can set up Wallarm to create issues in Jira when [vulnerabilities](../../../glossary-en.md#vulnerability) are detected, all or only for the selected risk level(s):
+* High risk
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/settings/integrations/jira.md:4]: add line break before the list

File: docs/latest/user-guides/settings/integrations/jira.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+##  Setting up integration
+
+In Jira UI: 
+
+1. Generate API token as described [here](https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/#Create-an-API-token).
+1. Copy the generated API token.
+
+In Wallarm UI:
+
+1. Open the **Integrations** section.
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/settings/integrations/jira.md:17]: Open Wallarm Console -> **Integrations** -> **JIRA**. - is enough to replace steps 1-2

File: docs/latest/user-guides/settings/integrations/jira.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+1. Perform the next steps in the **Incident and task management systems** section.
+1. Click the **Jira** block or click the **Add integration** button and choose **Jira**.
+1. Enter an integration name.
+1. Enter Jira host (e.g., `https://company-x.atlassian.net/`).
+1. Specify issuer email, which is required for Jira authentication, and also wll be used for the created issues as their author info.
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/settings/integrations/jira.md:22]: issuer email - to be fixed

File: docs/latest/user-guides/settings/integrations/jira.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+1. Paste the generated API token. Authentication data (host, email, token) will be checked.
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/settings/integrations/jira.md:23]: host is not auth data

The email and token will be used to authenticate Wallarm in JIRA available at the specified host.

File: docs/latest/user-guides/settings/integrations/jira.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+1. Select Jira space to create issues in. Available only on successful authentication.
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/settings/integrations/jira.md:24]: do we check auth data before a user clicks button "create integration"?

File: docs/latest/user-guides/settings/integrations/jira.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+--8<-- "../include/integrations/update-integration.md"
+
+## Disabling integration
+
+--8<-- "../include/integrations/disable-integration.md"
+
+## Deleting integration
+
+--8<-- "../include/integrations/remove-integration.md"
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/settings/integrations/jira.md:46]: 1. check that token value of the screenshot is test
2. Default JIRA task types are the following: Task, Bug. Please specify Bug on the screenshot
3. JIRA space also can be renamed to be more native for security area, e.g. Security
4. JIRA on the background is highlighted in a strange way - more area than needed

File: docs/latest/user-guides/settings/integrations/jira.md (hunk 1 above)
Comment (AnastasiaTWW) [inline docs/latest/user-guides/settings/integrations/jira.md:1]: let;s name Jira as Jira Atlassian for more results for user search input - they can use Atlassian as a search tag

File: docs/latest/user-guides/settings/integrations/jira.md (hunk 6 above)
Comment (MaximBashurov) [reply docs/latest/user-guides/settings/integrations/jira.md:24]: Yes.

File: docs/latest/user-guides/settings/integrations/jira.md [hunk 8]
```diff
@@ -14,13 +15,11 @@ In Jira UI:
...
-1. Open the **Integrations** section.
-1. Perform the next steps in the **Incident and task management systems** section.
-1. Click the **Jira** block or click the **Add integration** button and choose **Jira**.
+1. Open Wallarm Console → **Integrations** → **Jira**.
 1. Enter an integration name.
 1. Enter Jira host (e.g., `https://company-x.atlassian.net/`).
-1. Specify issuer email, which is required for Jira authentication, and also wll be used for the created issues as their author info.
-1. Paste the generated API token. Authentication data (host, email, token) will be checked.
+1. Enter the Jira user email, which Jira requires for authentication and also will be used to identify the Reporter for created issues.
+1. Paste the generated API token. The email and token will be checked to authenticate Wallarm in JIRA available at the specified host.
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/settings/integrations/jira.md:22]: Jira is correct

**Pull request merged**